providers/shared/components/apigateway/id.ftl

```diff
@@ -102,9 +102,15 @@ object.
                 "Default" : []
             },
             {
-                "Names" : "Authentication",
+                "Names" : ["AuthorisationModel", "AuthorizationModel", "Authentication"],
```

Comment:
Should probably include a description if we are making things easier to follow

Reply:
Will do

Reply:
Done